javascript/javascript1/week3/homework5/travelInfo.js

```diff
@@ -0,0 +1,15 @@
+const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  function getTripDuration(obj){
+     const durationInt = Math.floor(obj.destinationDistance / obj.speed *60);
+     const durationHrs = Math.floor(durationInt/60);
+     const durationMn = Math.floor(durationInt%60);
```

Comment:
Excellent implementation; very good use of remainder! 🤩 Just to be annoying, if something would take one minute, it would log `0hours and 1minutes`. Can you think of a way to improve that? 😄

Reply:
Thank you for the valuable input. the easiest fix, would be to put the "s" between () 😄. 
Otherwise I just turned the words "hours" and "minutes" into ternary operation depending on the value of hours and minutes.